src/components/OnBoarding/SecondForm.jsx

```diff
@@ -0,0 +1,134 @@
+import { useState } from 'react';
+import propTypes from 'prop-types';
+import { Box, FormControl, FormLabel, Input, Button, Flex, Text, HStack, Image, Stack, SimpleGrid } from '@chakra-ui/react';
+import FPH_LOGO from './fph_logo.png';
+
+const SecondForm = ({ handleChange, prevStep, nextStep }) => {
+  const [businessAddress1, changeBusinessAddress1] = useState('');
+  const [businessAddress2, changeBusinessAddress2] = useState('');
+  const [city, changeCity] = useState('');
+  const [state, changeState] = useState('');
+  const [postalCode, changePostalCode] = useState('');
+  const [country, changeCountry] = useState('');
```

Comment:
dont create these states, use the states from `formData` in here, such that when we press the back button, we don't lose all of our entries